model/task_lifecycle.go

```
@@ -134,6 +134,7 @@ func SetActiveState(t *task.Task, caller string, active bool) error {
 		}
 	}
 
+	UpdateBuildAndVersionStatusForTask(t)
```

We should return this so that we don't suppress the error